runtime/lib/ttnn/operations/eltwise/where/where.h

```diff
@@ -0,0 +1,15 @@
+// SPDX-FileCopyrightText: (c) 2024 Tenstorrent AI ULC
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef TTNN_RUNTIME_WHERE_H
```

Comment:
Include guards should be named based on the full path in a project's source tree to guarantee uniqueness, I think we even had a clang rule for this. See more here: https://google.github.io/styleguide/cppguide.html#The__define_Guard

I see that other headers in `runtime/lib/ttnn/operations` break this rule too. FYI @jnie-TT